### PR TITLE
(feat) O3-1649: Show vital signs without needing to expand the vitals header

### PR DESCRIPTION
**Pull request by jnsereko**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary

The vitals header design has been improved so that it is no longer collapsable. This ensures the most recent vitals for the patient are always visible.

Full design documentation for the vitals header [can be found here](https://zeroheight.com/23a080e38/p/61756c-vitals-header) 

Current & updated Vital Signs designs: https://zpl.io/aBPkgOO

cc @hadijahkyampeire @denniskigen @vasharma05 

## Screenshots

https://user-images.githubusercontent.com/58003327/219154235-45289df2-5f5d-4bd1-bbaf-1f322c8346ca.mov

### CODE REVIEW

**Review by hadijahkyampeire — COMMENTED** [review]
Thanks @jnsereko for the work, it looks good, could you please fix the failing test.

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header-item.scss [hunk 1]
```diff
@@ -12,7 +12,12 @@
 }
 
 .abnormal-value {
-  background-color: rgba(0, 0, 0, 0.1);
+  border: 2px solid $danger;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: fit-content;
+  padding: 0 5.7px 0 4px;
```
Comment (hadijahkyampeire) [inline packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header-item.scss:20]: nit: we should always prefer rem to px for consistence.

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.component.tsx [hunk 2]
```diff
@@ -89,38 +89,23 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid, showRecordVita
...
+              <ArrowRight
                 size={16}
-                className={styles['expand-button']}
-                title={'ChevronDown'}
-                onClick={(e) => {
-                  e.stopPropagation();
-                  toggleDetailsPanel();
-                }}
+                className={styles['arrow-up-button']}
+                style={{ fill: '#0f62fe' }}
```
Comment (hadijahkyampeire) [inline packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.component.tsx:95]: Why use an inline style when you have a class, you could add the fill style to the `arrow-up-button` class in the .scss file.

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.scss [hunk 3]
```diff
@@ -53,6 +54,12 @@
   border-bottom: 1px solid $color-yellow-50;
 }
 
+.container-row {
+  border-bottom: 1px solid #e0e0e0;
```
Comment (hadijahkyampeire) [inline packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.scss:58]:
```suggestion
  border-bottom: 1px solid #e0e0e0;
  border-bottom: 1px solid $grey-2;
```
We should use the variables from our color pallette so as to be sure of the uniformity of our colors.

**Review by hadijahkyampeire — COMMENTED** [review]
Very very close, Just a few minor tweaks.

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.component.tsx (hunk 2 above)
Comment (hadijahkyampeire) [reply packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.component.tsx:95]: Also the color variable for that code is `$color-blue-60-2`

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header-item.component.tsx [hunk 4]
```diff
@@ -14,20 +14,25 @@ const VitalsHeaderItem: React.FC<VitalsHeaderItemProps> = ({ interpretation, val
...
-      <div className={styles['value-container']}>
-        <label className={styles['pad-right']}>
-          <span className={styles.value}>{value || '-'} </span>
-          <span className={styles.units}>{value && unitSymbol}</span>
-        </label>
-        {interpretation === 'high' ? <span className={styles.high}></span> : null}
-        {interpretation === 'critically_high' ? <span className={styles['critically-high']}></span> : null}
-        {interpretation === 'low' ? <span className={styles.low}></span> : null}
-        {interpretation === 'critically_low' ? <span className={styles['critically-low']}></span> : null}
+    <div className={`${styles.container}`}>
```
Comment (denniskigen) [inline packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header-item.component.tsx:17]:
```suggestion
    <div className={styles.container}>
```

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header-item.scss [hunk 5]
```diff
@@ -12,7 +12,12 @@
 }
 
 .abnormal-value {
-  background-color: rgba(0, 0, 0, 0.1);
+  border: 2px solid $danger;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: fit-content;
+  padding: 0 0.25rem 0 0.25rem;
```
Comment (denniskigen) [inline packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header-item.scss:20]: nit: Shorthand

```suggestion
  padding: 0rem 0.25rem;
```

**Comment by denniskigen** [comment]
Thanks a bunch for the great work, @jnsereko!

**Review by denniskigen — APPROVED** [review]
Thanks a bunch for the great work, @jnsereko!

**Pull request merged**